wpeverest/everest-forms
# Feature - New "Range" option for Date

### All Submissions:






### Changes proposed in this Pull Request:



A new option called "Enable range" is added for Date/Time field's advanced field using which the user can select a series/range of dates for entry.
### How to test the changes in this Pull Request:

1. Create a form and tick the enable range in field option.
2. Add a date/time field.
3. Go to its advanced option and click on Enable Range.
4. Check if you can select a whole range of dates from the front end.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Introduction of range in the date/time field.
> Tweak   - Refactoring in the code of date/time field.
